dokku

```diff
@@ -159,11 +159,16 @@ case "$1" in
   help|'')
     echo "Usage: dokku [--quiet|--trace|--rm-container|--rm|--force] COMMAND <app> [command-specific-options]"
     echo ""
-    echo "Options:"
+    echo "Primary help options, type \"dokku COMMAND:help\" for more details, or dokku help --all to see all commands."
+    echo "Commands:"
```

Comment:
I think consolidating lines 162 and 163 to the following (which is an almost direct copy of heroku's msg) makes things clearer.
```
    echo "Primary help topics, type 'dokku TOPIC:help' for more details:"
```